src/topology/order.lean

```diff
@@ -294,6 +294,21 @@ lemma is_open_coinduced {t : topological_space α} {s : set β} {f : α → β}
   @is_open β (topological_space.coinduced f t) s ↔ is_open (f ⁻¹' s) :=
 iff.rfl
 
+lemma is_open_preimage_of_coinduced [topological_space α] [topological_space β] {π : α → β}
+  (h : ‹topological_space β› = topological_space.coinduced π ‹_›) {s : set β}
+  (hs : is_open s) : is_open (π ⁻¹' s) :=
+by rwa h at hs
+
+lemma preimage_mem_nhds_of_coinduced [topological_space α] [topological_space β] {π : α → β}
+  (h : ‹topological_space β› = topological_space.coinduced π ‹_›) {s : set β}
+  {a : α} (hs : s ∈ 𝓝 (π a)) : π ⁻¹' s ∈ 𝓝 a :=
+begin
+  rcases mem_nhds_sets_iff.mp hs with ⟨V, hVs, V_op, mem_V⟩,
+  rw mem_nhds_sets_iff,
+  exact ⟨π ⁻¹' V, set.preimage_mono hVs, is_open_preimage_of_coinduced h V_op,
+         by rwa ← hab at mem_V⟩
```

Comment:
This probably fixes the build:
```suggestion
  exact ⟨π ⁻¹' V, set.preimage_mono hVs, is_open_preimage_of_coinduced h V_op, mem_V⟩
```

Reply:
Sorry, I didn't see this message before fixing it.